src/core/RadioButton/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import useStyles from './styles';
+import { RadioBaseProps } from './base';
+import Radio from '@material-ui/core/Radio';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+interface RadioButtonProps extends RadioBaseProps {
+  children: string;
+  disabled?: boolean;
+  checked?: boolean;
+}
+
+const RadioButton: React.FC<RadioButtonProps> = ({
+  children,
+  disabled,
+  checked,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <FormControlLabel
+      value="end"
+      label={children}
+      className={classes.RadioLabel}
+      control={
+        <Radio
+          color="primary"
```

Comment:
We are having a `highlight` color here instead not `primary`. The API suggest it takes `secondary` as default and can be override with default, primary, secondary, which is not our use case. [https://material-ui.com/api/radio/](https://material-ui.com/api/radio/)